lm_eval/tasks/ammlu/_default_template_yaml

```diff
@@ -0,0 +1,19 @@
+group: ammlu
+dataset_path: Hennara/ammlu
+test_split: test
+fewshot_split: dev
+fewshot_config:
+  sampler: first_n
+output_type: multiple_choice
+doc_to_text: "{{Question.strip()}}\nA. {{A}}\nB. {{B}}\nC. {{C}}\nD. {{D}}\nالجواب："
```

Comment:
I'm aware that Arabic is read from right to left, is this doc_to_text format typically how it's done?

Reply:
for the doc_to_text, yes this how is gonna be format, but I forget to change the description
you are write the description should be in Arabic not in English, I will change that immediately

Reply:
Is this how the model's trained as well and are the tokens generated in reverse? :o